client/src/components/Workflow/Workflows.test.js

```diff
@@ -126,7 +136,7 @@ describe("WorkflowList.vue", () => {
                 })
                 .reply(200, mockWorkflowsData, { total_matches: "1" });
 
-            const tags = wrapper.findAll("tbody > tr .tag-name").wrappers;
+            const tags = wrapper.findAll(Tag).wrappers;
```

Comment:
Finding via component is preferable, as it's not prone to breaking due to template changes. The reason it didn't work is because the test was missing a import for the component.

Reply:
Done - in this [commit here](18d61f066a411ff3a5b3756123ee677b8b93dedf).